test/corpus/create.txt

```diff
@@ -1357,3 +1357,86 @@ CACHED IN 'pool1' WITH REPLICATION = 2
         (keyword_with)
         (keyword_replication)
         value: (literal)))))
+
+================================================================================
+Create table with string name
```

Comment:
Test Name might be misleading. In postgres double quotes just mean that the Case is kept (e.g. "uPPeRCase") otherwise postgres will lowercase all characters by default.